src/api/totals.js

```diff
@@ -13,5 +13,10 @@ module.exports = {
   findByPage: function(pageId, callback, options) {
     var params = _.merge({ page: pageId }, options);
     return getJSONP(routes.get('totals', params), callback);
+  },
+
+  findByCharity: function(charityUid, callback, options) {
```

Comment:
We created the totals.js file together the other day and forgot to add a test file - might be a good time to set that up and make sure these new methods have tests.